test/deprecated/io/warnings.chpl

```diff
@@ -40,12 +66,6 @@ stderr
 stdinInit // TODO: document?
```

Comment:
I'd alter these comment lines so it doesn't look like stdinInit, etc. still need to be tested.  Something like 
```
// TODO: document stdinInit?
```

Reply:
How will it be if I add these comments beside the called function? like
```
stdinInit();    // TODO: document?
```

Reply:
I like that, that also works!